lib/tokenFactory.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const BaseFactory = require('./baseFactory');
+const Token = require('./token');
+const iron = require('iron');
+const nodeify = require('./nodeify');
+// Symbols for private fields
+const password = Symbol('password');
+
+let instance;
+
+/**
+ * Seal token value
+ * @method sealToken
+ * @param  {String}         tokenValue     Token value to seal
+ * @param  {String}         pw             Password to seal with
+ * @return {Promise}
+ */
+function sealToken(tokenValue, pw) {
```

Comment:
This code is identical to that in `secretFactory`. Seems like an opportunity for refactoring.